components/03-modules/mega-menu/_mega-menu.scss

```diff
@@ -96,6 +114,16 @@ $mega-menu__inner-link-color--active         : $color-primary !default;
             flex-basis: 100%;
             justify-content: flex-end;
             padding: $mega-menu__inner-item-padding--image;
+
+            .mega-menu__inner-link {
+                display: block;
+                &:focus {
+                    @include focus();
+                    background-color: transparent;
+                    color: $font-color-base;
```

Comment:
Please move to variables on the top of this file

Reply:
done